samcli/cli/cli_config_file.py

```diff
@@ -57,14 +57,12 @@ def __call__(self, config_path, config_env, cmd_names):
 
         samconfig = SamConfig(config_file_dir, config_file_name)
 
-        # Enable debug level logging by environment variable "SAM_DEBUG"
-        if os.environ.get("SAM_DEBUG", "").lower() == "true":
-            LOG.setLevel(logging.DEBUG)
-
-        LOG.debug("Config file location: %s", samconfig.path())
-
-        if not samconfig.exists():
-            LOG.debug("Config file '%s' does not exist", samconfig.path())
+        # bringing samconfig file location up to info level,
+        # to improve UX and make it clear where we're looking for samconfig file
+        if samconfig.exists():
+            click.echo(f"Config file location: {samconfig.path()}")
+        else:
+            click.secho(f"Config file '{samconfig.path()}' does not exist", fg="yellow", err=True)
```

Comment:
The samconfig is not necessary in the project folder. I would prefer not to put it in error level.

Reply:
agree

Reply:
updated